### ISSUE TITLE
Enable USB debugging

### ISSUE THREAD

**Issue by mjray** [tranquvis/SimpleSmsRemote#20]
Would you add a command to enable USB debugging, please?

That would make this app more helpful as disaster recovery preparation, giving an easier way to get into devices with destroyed touch or screens but no way to connect external screens/pointers.

It wouldn't be much use to a remote attacker if it's only USB not TCP because they'd need physical access to the device, as well as spoofing a correct sending number and knowing the syntax - and ultimately, if an attacker has physical access, it's probably game over anyway.

**Comment by tranquvis** [comment]
Hello @mjray ,

Please correct me if I got it wrong.

You want to enable android usb-debugging via sms.
This is only possible with root access.

I decided to don't implement root access features because I'm not sure about how to implement and how much time this would consume.

**Comment by mjray** [comment]
Yes, that's what I'm looking for. I don't know how to implement root access either yet. Maybe a good samaritan will rescue us one day :)

**Comment by aRTee42** [comment]
I just started playing around with simplesmsremote, and with similar programs. Macrodroid threw an error in the logfile due to not being able to turn on data.
However, instead of mentioning it needs root, it told me to use ADB with the command:
"adb shell pm grant com.arlosoft.macrodroid android.permission.WRITE_SECURE_SETTINGS"
I tried it, and it worked fine, the program could toggle on and off the data connection.

So I figured, I try the same with simplesmsremote:
"adb shell pm grant tranquvis.simplesmsremote android.permission.WRITE_SECURE_SETTINGS"
to which the phone unfortunately responded with:
"Operation not allowed: java.lang.SecurityException: Package tranquvis.simplesmsremote has not requested permission android.permission.WRITE_SECURE_SETTINGS"

This on a side note (what I really want is to be able to togge data via sms - my wife forgets to turn on data all the time, then I can't send her Signal messages), but it may come in handy if the same/similar permissions as the one above are helpful for that.
